lineQueueClient/src/components/Signup/Signup.style.js

```diff
@@ -1,33 +1,58 @@
-import { StyleSheet } from 'react-native';
+import {StyleSheet, Dimensions} from 'react-native';
+
+let width = Dimensions.get('window').width
+let height = Dimensions.get('window').height
 
 const signupStyle = StyleSheet.create({
-  containerWhite: {
-    flex: 1,
+  contentContainer: {
+    display: 'flex',
     flexDirection: 'column',
-    justifyContent: 'center',
+    justifyContent: 'space-around',
+    alignItems: 'stretch',
+    backgroundColor: 'white',
+    flex: 1
+  },
+  inputContainer: {
+    flex: 1,
     alignItems: 'center',
-    backgroundColor: 'white'
+    justifyContent: 'center',
   },
-  text: {
-    fontSize: 16,
-    textAlign: 'center',
-		fontFamily: 'Assistant',
-    color: '#151515'
+  inputContainerBottom: {
```

Comment:
rename to `buttonContainer`

Reply:
buttonContainer was already used for the Button Box. Renamed inputContainerButtom to buttonContainer and  what was buttonContainer to buttonBox